src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -61,4 +83,38 @@ public CreateCaseCallbackService(
             .filter(item -> !Strings.isNullOrEmpty(item.getTransformationUrl()))
             .getOrElse(Validation.invalid("Transformation URL is not configured"));
     }
+
+    private Validation<Seq<String>, Map<String, Object>> createNewCase(
```

Comment:
What does the sequence represent in the validation? Is it errors or warnings?

Reply:
As per current design orchestrator is using - it is always responding with errors. Warnings are not used. Transformation will return warnings on successful response so this would mean successful response + warnings. slightly different behaviour. Errors + warnings never used nor distinguished. Nor adapted in orchestrator callback at any point in the past

Reply:
Transformation API can return a combination of errors and warnings so that they can be presented to the caseworker as such. We can't ignore warnings.

Note that not all calls to our create case API will result in a new case - it depends on `ignoreWarnings` flag. On the first attempt, if there are warnings, we want to present them to the caseworker and let them decide what to do next.

Errors and warnings have been working together in attaching exception record to a case.

Reply:
> Errors and warnings have been working together in attaching exception record to a case.

I don't see warnings being there:
https://github.com/hmcts/bulk-scan-orchestrator/blob/0ec25b7f8601f7d36e197d6e7884642fbe6f3ab4/src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CcdCallbackController.java#L42-L57

it's always an error

transformation api either throws an exception or returns case details with warnings. that is / should be handled in next pr. or perhaps include in the mapping - at least that's what ccd api branch currently does

Reply:
Regarding attaching exception records, you're right - in practice we don't provide values for warnings here. I thought we did at some point, remembering that we tested whether we could see both at the same time, but looks like I was wrong about adding any warnings. But it doesn't matter - we're building a new endpoint, which has its own requirements.

> transformation api either throws an exception or returns case details with warnings. that is / should be handled in next pr. or perhaps include in the mapping - at least that's what ccd api branch currently does

This is what our java client might be doing at the moment, but transformation API returns either case details with warnings or errors with warnings, as per spec: https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1064666568#TechnicalprerequisitesandinformationforServiceon-boardingwithPhase2Bulkscanning-2.4.5OpenAPI(swagger)specificationforexceptionrecordtransformationAPI

We need to reflect that in our responses to CCD.

Reply:
ok. Can I suggest return object TBD in the future then and leave this as is? I do have warnings accompanied later on which I was meant to discuss with the team when reaching such point. But I think this counts as a discussion and I'm aware of future development now

Reply:
Up to you. My recommendation, given it's a new method and we know that it won't look like this when the story is complete, is to prepare dependencies first:
- adjustment of transformation client, so that we can distinguish between 422/400
- preparation of the part where we talk to CCD to create the case

Then you could easily complete this one without having to stub things and leaving it for further modifications.